### PR TITLE
LPS-133314 accessibility search portlet

### PR DESCRIPTION
**Pull request by georgel-pop-lr**
For more details, check description of LPS-133314

### CODE REVIEW

**Review by kresimir-coko — APPROVED** [review]
LGTM! 👍

**Comment by georgel-pop-lr** [comment]
> LGTM!

Hi @kresimir-coko,

Can we merge this, or we wait for someone else review?

Regards.

**Comment by kresimir-coko** [comment]
I'd expect someone else to review before we're ready to forward. Perhaps @pat270 for the CSS stuff or @markocikos for the generic overview.

**Comment by georgel-pop-lr** [comment]
> I'd expect someone else to review before we're ready to forward. Perhaps @pat270 for the CSS stuff or @markocikos for the generic overview.

Okay, sounds good to me, thanks

File: modules/apps/message-boards/message-boards-parser-bbcode/src/main/java/com/liferay/message/boards/parser/bbcode/internal/HtmlBBCodeTranslatorImpl.java [hunk 1]
```diff
@@ -293,13 +293,13 @@ protected void handleCode(
 
 		for (int i = 0; i < lines.length; i++) {
 			sb.append("<tr>");
-			sb.append("<td class=\"line-numbers\" data-line-number=\"");
+			sb.append("<td class=\"line-numbers\">");
```
Comment (markocikos) [inline modules/apps/message-boards/message-boards-parser-bbcode/src/main/java/com/liferay/message/boards/parser/bbcode/internal/HtmlBBCodeTranslatorImpl.java:296]: Minor SF: These two lines can be merged into one `append`.

File: modules/apps/message-boards/message-boards-parser-bbcode/src/main/java/com/liferay/message/boards/parser/bbcode/internal/HtmlBBCodeTranslatorImpl.java [hunk 2]
```diff
@@ -293,13 +293,13 @@ protected void handleCode(
...
 
 			String index = String.valueOf(i + 1);
 
 			sb.append(index);
 
-			sb.append("\"></td>");
+			sb.append("</td>");
 			sb.append("<td class=\"lines\">");
```
Comment (markocikos) [inline modules/apps/message-boards/message-boards-parser-bbcode/src/main/java/com/liferay/message/boards/parser/bbcode/internal/HtmlBBCodeTranslatorImpl.java:303]: Minor SF: These two lines can be merged into one `append`.

File: portal-impl/src/com/liferay/portal/servlet/ComboServlet.java [hunk 3]
```diff
@@ -390,6 +390,13 @@ protected void doService(
 
 					baseURL = PortalUtil.getPathProxy() + baseURL;
 
+					if (StringUtil.contains(
+							stringFileContent, _CSS_CHARSET_UTF_8, "")) {
```
Comment (markocikos) [inline portal-impl/src/com/liferay/portal/servlet/ComboServlet.java:394]: Minor SF: We can use `StringPool.BLANK` here

**Review by markocikos — COMMENTED** [review]
@georgel-pop-lr few minor SF nits inline, otherwise LGTM!

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
> @georgel-pop-lr few minor SF nits inline, otherwise LGTM!

Hi @markocikos,

All the requested changes done, please review. 

Regards.

**Review by markocikos — APPROVED** [review]
LGTM!

**Comment by markocikos** [comment]
ci:forward

**Pull request closed without merging**